predeploy.sh

```diff
@@ -2,22 +2,14 @@
 
 # === clean up old state ===
 
-# client
-rm -rf dist
-
-# server
+# server-side
 rm -rf functions/content
 rm -rf functions/.nuxt/*
 rm -rf functions/server-middleware
 
 # === set up new state ===
 
-# client
-mkdir dist
-cp -r .nuxt/dist/client dist
-cp -r static/* dist
-
-# server
+# server-side
 cp -r content functions/content
```

Comment:
So when you `generate`, you still need the content on the server?what you mean by the server being an api for content.... what is `generate` doing, and why is there no `dist` anymore?